### PR TITLE
add stale bot

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

/kind cleanup

**What this PR does / why we need it**:
Now issues reach ~200, close one by one by hand is bad.

The stale bot needs person who has write access to install, details see [this](https://github.com/marketplace/stale).

### CODE REVIEW

**Comment by kevin-wangzefeng** [comment]
Good idea. And we can check close out dated issues in parallel in the following days.

**Comment by daixiang0** [comment]
Move to prow stale, details refer to [config](https://github.com/kubernetes/test-infra/blob/5f26eecd5a377bb9e9c4a86d9b25710ff154d8f0/config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml).

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - reminder
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+# Label to use when marking as stale
+staleLabel: stale
```
Comment (kevin-wangzefeng) [inline .github/stale.yml:21]:
```suggestion
staleLabel: lifecycle/stale
```

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  daysUntilStale: 30
+  markComment: >
+    Hello 👋 Looks like there was no activity on this amazing PR for last 30 days.
+
+    **Do you mind updating us on the status?** Is there anything we can help with? If you plan to still work on it, just comment on this PR or push a commit. Thanks! 🤗
+
+    If there will be no activity for next week, this issue will be closed (we can always reopen a PR if you get back to this!).
+  #unmarkComment: No need for unmark comment.
+  closeComment: >
+    Closing for now as promised, let us know if you need this to be reopened! 🤗
```
Comment (kevin-wangzefeng) [inline .github/stale.yml:34]: Recommended to explicitly mention "daysUntilClose" in the closeComment.

**Comment by daixiang0** [comment]
Set daysUtilClose as 60d and daysUntilStale as 90d, keep sync with k8s does.

**Comment by daixiang0** [comment]
ping @kevin-wangzefeng @fisherxu

**Pull request merged**